ui/src/controller/track_controller.ts

```diff
@@ -277,6 +374,91 @@ export abstract class TrackController<
       }
     }
   }
+
+  /**
+   * Provide a non-viable Data serving as a token of cancellation of
+   * the bounds-change calculation.
+   *
+   * @return {Data} a token track-data object signalling cancellation
+   */
+  protected cancelData(): Data {
+    return {[cancelToken]: cancelToken} as unknown as Data;
```

Comment:
Nope, that was it, avoiding a larger refactoring. I happen to know that this will be used only in initialization of a controller for a track that is being filtered out, so the broken `Data` object will not matter. I should add some further explanatory comments to make sure that this will not be used in any other context. Thanks for raising this!

Reply:
Actually @jfaltermeier on reflection you are absolutely correct that we do not need this special cancellation token. I've cancelled it in commit 75ba5c5508ecf82d0d3a836db49de87053799773 😀  There is a perfectly reasonable empty data object that we can publish for both kinds of tracks involved in this PR.